Fill.py

```diff
@@ -839,23 +841,18 @@ def failed(warning: str, force: typing.Union[bool, str]) -> None:
 
     world_name_lookup = multiworld.world_name_lookup
 
-    block_value = typing.Union[typing.List[str], typing.Dict[str, typing.Any], str]
+    block_value = typing.Union[typing.List[str], typing.Dict[str, typing.Any]]
```

Comment:
Is this supposed to be an `=` ?

Reply:
This is a type alias.
The way to define type aliases has changed multiple times since Python 3.8;
https://docs.python.org/3/library/typing.html#type-aliases

It should probably be `BlockValue` instead of `block_value` to make it look like other user-defined types, and it should be in the module scope for type-checkers to recognize it as a type alias.